test/common/grpc/codec_test.cc

```diff
@@ -106,6 +106,59 @@ TEST(GrpcCodecTest, decodeInvalidFrame) {
   EXPECT_EQ(size, buffer.length());
 }
 
+// TODO: Should this test really pass? This test shows that null bytes in the bytestring
```

Comment:
Not sure if unassigned TODO is worth it, I would just move the question to after the explanation of what's going on.
Whenever someone decides to look into it, they'll see the test. 

Thank you for explaining this so clearly!

Reply:
Moved